docs/Other/marketplace_webhooks.md

```diff
@@ -774,6 +774,14 @@ title: Customer Delete Event
 
 The spend change request webhook is called every time a spend change is requested for one of the vendor's products.
 
+Details:
+
+- **currency** _The currency in which the spend/budget is being requested._
+- **requested_value** _The requested new spend value/budget, in cents (or smallest denomination in other currencies)._
+- **conversion_currency** _The currency that the original spend/budget was converted to._
+- **conversion_value** _The original requested new spend value/budget in cents, before currency conversion._
+- **conversion_rate** _The conversion rate used to convert ConversionValue to obtain the RequestedValue._

```

Comment:
I think this will make it a little more clear that the message is talking about the fields.
```suggestion
- **conversion_rate** _The conversion rate used to convert `conversion_value` to obtain the `requested_value`._

```